examples/kubernetes/kernel-check/kernel-check.yaml

```
@@ -0,0 +1,29 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: bpf-kernel-check
+  name: bpf-kernel-check
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: bpf-kernel-check
+  template:
+    metadata:
+      labels:
+        app: bpf-kernel-check
+    spec:
+      nodeSelector:
+      # run deployment on specific node, e. g. nodename:cilium
+      containers:
+        - name: cilium
+          image: docker.io/cilium/cilium:stable
+          command: ["sh", "-c", "cilium kernel-check"]
```

nit: Is the shell required here? Could this do something like this instead?

```suggestion
          command: ["cilium",  "kernel-check"]
```

---

you are right: there is no dedicated need for having a shell here. fixed.